db/scripts/cleanup_scheduled_calls.js

```diff
@@ -0,0 +1,27 @@
+async function getDb() {
+  const dotenv = require("dotenv");
+  dotenv.config();
+
+  const pgp = require("pg-promise")({});
+  const connectionURL = process.env.URI;
+  return pgp({
+    connectionString: connectionURL,
+    ssl: { rejectUnauthorized: false },
+  });
+}
+
+async function cleanupScheduledCalls() {
+  const db = await getDb();
```

Comment:
I reckon you could use the `AppContainer` here, rather than building the connection above

Reply:
This results in `SyntaxError: Cannot use import statement outside a module`. Most likely because the script is being run as is and not transpiled